crates/router/src/routes/app.rs

```diff
@@ -1020,7 +1020,11 @@ impl User {
                     web::resource("/verify_email_request")
                         .route(web::post().to(verify_email_request)),
                 )
-                .service(web::resource("/user/resend_invite").route(web::post().to(resend_invite)));
+                .service(web::resource("/user/resend_invite").route(web::post().to(resend_invite)))
+                .service(
+                    web::resource("/activate_from_email")
```

Comment:
I'd prefer `accept_invite_from_email`

Reply:
Updated route and function names as well